docs/config/cluster.rst

```diff
@@ -1415,6 +1415,41 @@ The following settings can be used to configure the behavior of the
   all data nodes are started. However, the value must be bigger than the half
   of the expected number of data nodes in the cluster.
 
+Logical Replication
+-------------------
+
+Replication process can be configured by the following settings. Settings
+are dynamic and can be changed in runtime.
+
+.. _replication.logical.ops_batch_size:
+
+**replication.logical.ops_batch_size**
+  | *Default:* ``50000``
+  | *Min value:* ``16``
+  | *Runtime:* ``yes``
+
+Maximum number of operations to replicate from the publisher cluster per poll.
+Represents a number to advance a sequence.
+
+.. _replication.logical.reads_poll_duration:
+
+**replication.logical.reads_poll_duration**
+  | *Default:* ``50``
+  | *Runtime:* ``yes``
+
+The maximum time (in milliseconds) to wait for changes per poll operation. When
+a subscriber makes another one request to a publisher, it has
+``reads_poll_duration`` milliseconds to harvest changes from the publisher.
+
+.. _replication.logical.indices.recovery.chunk_size:
+
+**replication.logical.indices.recovery.chunk_size**
```

Comment:
I'd suggest to rename it:
```suggestion
**replication.logical.recovery.chunk_size**
```

Reply:
Done in https://github.com/crate/crate/pull/12142/commits/1745ec128428b8ae7f191f14224fed1a165f8594